lib/vanity/templates/vanity.css

```diff
@@ -1,3 +1,5 @@
+.vanity { width: 90%; margin: 2em auto; font-family: "Helvetica Neue", "Helvetica", "Verdana", sans-serif }
```

Comment:
This change to almost-full width and other style changes would also be great to pull into a separate PR.